DatadogRUM/Sources/RUMConfiguration.swift

```diff
@@ -210,6 +210,14 @@ extension RUM {
         /// Default: `nil`.
         public var customEndpoint: URL?
 
+        /// Determines whether automatic tracking of anonymous user id is enabled.
+        ///
+        /// If enabled, the SDK will automatically track the anonymous user id for each RUM Session, which
+        /// is used to link RUM Sessions belonging to the same anonyomus user.
+        ///
+        /// Default: `true`.
+        public var trackAnonymousUser: Bool
```

Comment:
suggestion/ I'd consider renaming `trackAnonymousUser` to `trackAnonymousUserID` to emphasize that we’re persisting and using only ID, **rather than tracking any personal user data**. This explicit naming could also helps clarify that the data is purely non-personal, which might avoid confusion and potential privacy concerns. WDYT?

Reply:
Valid point, but AFAIK that's the API that was used in the browser.

Let me check if we can make an alignment.

If not - I'll make the docs more specific.

Reply:
It's already out on browser under the same name:
https://docs.datadoghq.com/real_user_monitoring/browser/setup/client/?tab=rum#configuration